app/test/integration/http/setup.dart

```diff
@@ -2,7 +2,7 @@ import 'dart:convert';
 import 'dart:io';
 
 import 'package:cookie_jar/cookie_jar.dart';
-import 'package:dio_retry/dio_retry.dart';
+//import 'package:dio_retry/dio_retry.dart';
```

Comment:
could you add a todo here?

TODO: add retry back after the compatibility issue is resovled
https://github.com/aloisdeniel/dio_retry/issues/10

(guessing I will just fork and fix it myself for now...)